### PR TITLE
Document list of permissions

### PR DESCRIPTION
**Pull request by Cisphyx**

### CODE REVIEW

File: docs/synapse/devguides/devops_cell.rst [hunk 1]
```diff
@@ -100,3 +100,313 @@ The generic Cell server can also be used for starting the Cell by specifying the
 
     python -m synapse.servers.cell path.to.MyCell /path/to/dirn
 
+Cell Permissions
+----------------
+
+The following permissions exist for controlling access to Cell operations.
+
+*admin.cmds*
```
Comment (vEpiphyte) [inline docs/synapse/devguides/devops_cell.rst:108]: This is a Cortex permission.

File: docs/synapse/devguides/devops_cell.rst [hunk 2]
```diff
@@ -100,3 +100,313 @@ The generic Cell server can also be used for starting the Cell by specifying the
...
+*auth.user.set.locked*
+    Set the locked status of a User.
+
+*auth.user.set.passwd*
+    Set a User's password.
+
+*auth.user.set.rules*
+    Set rules for a User.
+
+*axon.get*
```
Comment (vEpiphyte) [inline docs/synapse/devguides/devops_cell.rst:147]: axon perm

File: docs/synapse/devguides/devops_cell.rst [hunk 3]
```diff
@@ -100,3 +100,313 @@ The generic Cell server can also be used for starting the Cell by specifying the
...
+    Get any Storm Service definition.
+
+*service.get.<name>*
+    Get a specific Storm Service definition.
+
+*service.list*
+    List the Storm Service definitions.
+
+*status*
+    Get status information for a Cortex.
```
Comment (vEpiphyte) [inline docs/synapse/devguides/devops_cell.rst:367]: this is a Cell perm, not a cortex perm

Comment (vEpiphyte) [reply docs/synapse/devguides/devops_cell.rst:367]: nvm this is a deprected cortex perm

File: synapse/lib/stormtypes.py [hunk 4]
```diff
@@ -212,7 +212,7 @@ async def _libPkgAdd(self, pkgdef):
         Returns:
             dict: The validated storm package definition.
         '''
-        self.runt.user.confirm(('pkgs', 'add'), None)
+        self.runt.user.confirm(('pkg', 'add'), None)
```
Comment (vEpiphyte) [inline synapse/lib/stormtypes.py:215]: ????

Comment (Cisphyx) [reply synapse/lib/stormtypes.py:215]: storm package add/remove permissions were 'pkg' in cortex.py and 'pkgs' here, which I'm assuming was not intentional.

File: docs/synapse/devguides/cell_permissions.rst [hunk 5]
```diff
@@ -0,0 +1,331 @@
...
+    Get any Storm Service definition.
+
+*service.get.<name>*
+    Get a specific Storm Service definition.
+
+*service.list*
+    List the Storm Service definitions.
+
+*status*
+    Get status information for a Cortex.
```
Comment (vEpiphyte) [inline docs/synapse/devguides/cell_permissions.rst:292]: Note: This is currently a deprecated permission.

Comment (Cisphyx) [reply docs/synapse/devguides/cell_permissions.rst:292]: note added

**Pull request merged**